practicesharp/Equation_Validator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    class Equation_Validator
+    {
+        public string equation_s { get; private set; }
+
+        public Equation_Validator(string userinput)
+        {
+            equation_s = "";
+            if (Validate(userinput)) equation_s= userinput;
+        }
+        public bool Validate(string word)
+        {
+            try
+            {
+                const string allowed_letters = "1234567890+-*/";
+                foreach (char letter in word)
+                {
+                    if (allowed_letters.Contains(letter))
+                        continue;
+                    throw new System.IO.IOException();          //if letter is not allwoed then there's an input error.
```

Comment:
First of all, why no just write to console and return "false"?
The less complex your code, the better, always think about how you reduce complexity. 
Secondly, an exception should be used when an unexpected thing happens.
This function should just check if the string is a legit equation, if not return false, no exception should be thrown here.